code/modules/mob/living/basic/space_fauna/paper_wizard/paper_wizard.dm

```diff
@@ -0,0 +1,196 @@
+/mob/living/basic/paper_wizard
+	name = "Mjor the Creative"
+	desc = "A wizard with a taste for the arts."
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	faction = list(FACTION_HOSTILE, FACTION_STICKMAN)
+	icon = 'icons/mob/simple/simple_human.dmi'
+	icon_state = "paperwizard"
+	gender = MALE
+
+	response_help_continuous = "brushes"
+	response_help_simple = "brush"
+	response_disarm_continuous = "pushes"
+	response_disarm_simple = "push"
+	basic_mob_flags = DEL_ON_DEATH
+
+	maxHealth = 1000
+	health = 1000
+	melee_damage_lower = 10
+	melee_damage_upper = 20
+	obj_damage = 50
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+	ai_controller = /datum/ai_controller/basic_controller/paper_wizard
+	///spell to summon minions
+	var/datum/action/cooldown/spell/conjure/wizard_summon_minions/summon
+	///spell to summon clones
+	var/datum/action/cooldown/spell/pointed/wizard_mimic/mimic
+	///the loot we will drop
+	var/static/list/dropped_loot = list(/obj/effect/temp_visual/paperwiz_dying)
+
+
+/mob/living/basic/paper_wizard/Initialize(mapload)
+	. = ..()
+	apply_dynamic_human_appearance(src, mob_spawn_path = /obj/effect/mob_spawn/corpse/human/wizard/paper)
+	grant_abilities()
+	grant_loot()
+	RegisterSignal(src, COMSIG_MOVABLE_PRE_MOVE, PROC_REF(animate_step))
+
+/mob/living/basic/paper_wizard/proc/grant_abilities()
+	summon = new(src)
+	summon.Grant(src)
+	ai_controller.set_blackboard_key(BB_WIZARD_SUMMON_MINIONS, summon)
+	mimic = new(src)
+	mimic.Grant(src)
+	ai_controller.set_blackboard_key(BB_WIZARD_MIMICS, mimic)
+
+/mob/living/basic/paper_wizard/proc/grant_loot()
+	AddElement(/datum/element/death_drops, dropped_loot)
+
+/mob/living/basic/paper_wizard/proc/animate_step()
+	SIGNAL_HANDLER
+	var/paper_effect = new /obj/effect/temp_visual/paper_scatter(get_turf(src))
+	animate(paper_effect, alpha = 0, 1 SECONDS)
```

Comment:
It would probably make more sense for the effect to animate itself
Also, why does it animate over 1 second when the paper only exists for half a second?